diff --git a/CMakeLists.txt b/CMakeLists.txt
index c95b5fdf6..b661f39b4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -125,6 +125,7 @@ ENDIF()
 # Only for Unix: use pkg-config to find libraries
 # ===================================================
 INCLUDE(FindPkgConfig OPTIONAL)
+find_package(NumPy REQUIRED)
 # "QUIET" was introduced in 2.8.2 for PKG_CHECK_MODULES
 if (${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}.${CMAKE_PATCH_VERSION} VERSION_GREATER 2.8.1)
    set(_QUIET QUIET)
@@ -331,7 +332,7 @@ include(cmakemodules/script_create_version_h.cmake REQUIRED)  # Build version.h
 # ---------------------------------------------------------------------------
 INCLUDE_DIRECTORIES(".")
 INCLUDE_DIRECTORIES("${MRPT_CONFIG_FILE_INCLUDE_DIR}")
-
+INCLUDE_DIRECTORIES(${PYTHON_NUMPY_INCLUDE_DIR})
 # Add user supplied extra options (optimization, etc...)
 SET(USER_EXTRA_CPP_FLAGS "" CACHE STRING "Put extra compiler options here if desired")
